distribution/rpm/vector.spec

```diff
@@ -66,6 +66,7 @@ getent passwd %{_username} > /dev/null || \
     --comment "Vector observability data router" %{_username}
 chown %{_username} %{_sharedstatedir}/%{_name}
 usermod -aG systemd-journal %{_username}  || true
+usermod -aG systemd-journal-remote %{_username}  || true
```

Comment:
This was added by a contributor to the deb package, mirrored here for completeness.